packages/two_dimensional_scrollables/test/tree_view/tree_test.dart

```diff
@@ -718,6 +718,101 @@ void main() {
       expect(treeView.treeNodeBuilder, isA<TreeViewNodeBuilder<String>>());
       expect(treeView.treeRowBuilder, isA<TreeViewRowBuilder<String>>());
     });
+
+    testWidgets(
+        'TreeSliverNode should close all children when collapsed when animation is completed',
```

Comment:
Can you update the language here? This seems to be referring to the 1D version in the framework.